<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OVM_Dialog</class>
 <widget class="QWidget" name="OVM_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Optic variation method (orbit flattening)</string>
  </property>
  <layout class="QGridLayout" columnstretch="1,1">
   <item row="0" column="0">
    <widget class="QGroupBox">
     <property name="title">
      <string>Monitor readouts:</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="TableView" name="mon_tab"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox">
     <property name="title">
      <string>Target values:</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="TableView" name="con_tab"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox">
     <property name="title">
      <string>Calculated steerer corrections</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="TableView" name="var_tab">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="update_fit_button">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="execute_corrections">
          <property name="text">
           <string>Upload</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>madgui/widget/tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
